notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>This tutorial demonstrates how you can use Vertex AI Tabular Workflow for Forecasting to train an AutoML model. You can choose between the following model types: Time Series Dense Encoder (TiDE), Learn to Learn (L2L), Sequence to Sequence (Seq2Seq+), and Temporal Fusion Transformer (TFT). </p><p> </p><p>The function name is get_learn_to_learn_forecasting_pipeline_and_parameters(), not get_l2l_forecasting_pipeline_and_parameters(), right? </p><p> </p><p>Are we already supporting probabilistic inference?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Nice catch, updated to <code>get_learn_to_learn_forecasting_pipeline_and_parameters</code> .</p><p> </p><p>For probabilistic inference, yes, it is already supported in L2L and TiDE. </p>

Reply:
<p>Fixed.</p>